.github/workflows/ci.yml

```diff
@@ -68,16 +68,9 @@ jobs:
         run: bundle exec rake rubocop
   features:
     needs: [spec, spec-legacy]
-    runs-on: macos-12
+    runs-on: ubuntu-latest
```

Comment:
🗒️ Previously we had to run the Cucumber job on a macOS runner. This was a hack because macOS was the only runner able to run VirtualBox, and in the past year, this has become less and less reliable, to the point where VirtualBox would almost always hang or fail to start.

Now that we are using the Docker image to run the sshd container instead of VirtualBox, we can use a standard Ubuntu runner, which includes reliable Docker support.